custom-completions/git/git-completions.nu

```diff
@@ -161,3 +161,15 @@ export extern "git cherry-pick" [
   --abort                                       # Cancel the operation
   --skip                                        # Skip the current commit and continue with the rest of the sequence
 ]
+
+# Rebase the current branch
+export extern "git rebase" [
+  branch?: string@"nu-complete git switchable branches"  # name of the branch to rebase onto
+  upstream?: string@"nu-complete git commits"            # upstream branch to compare against
```

Comment:
I think both positions support commits as well as branches as their refspec.

Reply:
Ah yes, thanks! :pray:  I opted for adding another nu command which concatenates the output of switchable branches with the commits. To keep the helpful description for the commits, I added the default `Branch` description to the branches. Screenshot attached to make it clearer ^^

Would that work?

![Branches_And_Commits](https://user-images.githubusercontent.com/47532708/224539602-e447a7e0-18d9-4129-bc55-dffbcb063414.png)

Reply:
Awesome looks good!